packages/frontend/src/state/ethPriceCharts/atoms.ts

```diff
@@ -409,3 +410,24 @@ export const useLongChartData = () => {
     },
   )
 }
+
+export const useCrabPnLV2ChartData = () => {
+
+
+  return useQuery(
+    ['pnlChart', ],
+    async () => getCrabPnlV2ChartData(),
+    {
+      staleTime: Infinity,
+      refetchOnWindowFocus: true,
+    },
+  )
+}
+
+export const crabV2DaysAtom = atom(getCrabStartDate())
+
+function getCrabStartDate() {
```

Comment:
What does this function do? If it's going to return 30 always, we can replace it with a constant

Reply:
Removed for now since the feature is not implemented...will add as a constant when i want to implement the feature intended